### DISCUSSION TITLE
CallPeaks macs2 redundant rate

### DISCUSSION THREAD

**Discussion by Dena2021** [stuart-lab/signac#1243] [Q&A] [answered]
I ran CallPeaks using macs on my clusters
macspeaks <- CallPeaks(mySeurat, macs2.path = "/apps/RH7U2/gnu/python/2.7.13/bin/macs2",group.by = 'cluster.treat',cleanup=FALSE, outdir = "macs_peaks_dir")

I am concerned in regards to the high redundant rate reported in the macs xls output. I speculate that using the fragment file the nuclei barcodes are not accounted for, meaning, fragments that have the same genome coordinates are considered redundant. Is this correct? 

See header lines in the peaks.xls file: 

 This file is generated by MACS version 2.1.1.20160309
 Command line: callpeak -t /tmp/RtmpLp5gHn/0_MUT.bed -g 2.7e+09 -f BED --nomodel --extsize 200 --
shift -100 -n 0_MUT --outdir macs_peaks_dir
 ARGUMENTS LIST:
 name = 0_MUT
 format = BED
 ChIP-seq file = ['/tmp/RtmpLp5gHn/0_MUT.bed']
 control file = None
 effective genome size = 2.70e+09
 band width = 300
 model fold = [5, 50]
 qvalue cutoff = 5.00e-02
 Larger dataset will be scaled towards smaller dataset.
 Range for calculating regional lambda is: 10000 bps
 Broad region calling is off
 Paired-End mode is off

 tag size is determined as 177 bps
 total tags in treatment: 26012955
 tags after filtering in treatment: 16224659
 maximum duplicate tags at the same position in treatment = 1
 Redundant rate in treatment: 0.38
 Sequencing ends will be shifted towards 5' by 100 bp(s)
 d = 200

Thanks :)

**Comment by timoast** [comment, marked as answer]
The cell barcodes are not taken into account by MACS2, that is correct
